### PR TITLE
[Pr 134] adding prices to gear summary on share page

### PR DESCRIPTION
**Pull request by crsmyz**
adding prices to gear summary on share page.

### CODE REVIEW

File: server/views.js [hunk 1]
```diff
@@ -365,7 +365,7 @@ const renderCategory = function (category, args) {
     category.subtotalWeightDisplay = weightUtils.MgToWeight(category.subtotalWeight, args.totalUnit);
 
     category.subtotalPriceDisplay = category.subtotalPrice ? category.subtotalPrice.toFixed(2) : '0.00';
-    
+
```
Comment (crsmyz) [inline server/views.js]: Will remove this space when I revisit this PR

**Comment by galenmaly** [comment]
Thanks for tackling this!

Looks like we need to add the cells for the consumable/worn/base weight rows if those are visible.

**Comment by crsmyz** [comment]
@galenmaly can you explain where you'd want that or where its not showing up, right now I see it both on the share page and the normal page as soon as some marks consumables/worn next to an item

**Comment by galenmaly** [comment]
Yep - when items in a list are marked as consumable or worn, they'll add a new row to the totals table. I'm seeing some missing cells causing mis-alignments in this case:

prices-on-share

**Comment by crsmyz** [comment]
@galenmaly okay those new template changes should fix the spacing issue

**Comment by galenmaly** [comment]
Thanks for fixing the issue! Looks great! 🚀

**Pull request merged**